<launch>
<arg name="ns"/>

<include file="$(find px4)/launch/multicopter_x.launch">
	<arg name="ns" value="$(arg ns)"/>
</include>

<group ns="$(arg ns)">
	<param name="MPP_XY_P" type="double" value="1.0" />
	<param name="MPP_XY_FF" type="double" value="0.1" />
	<param name="MPP_XY_VEL_P" type="double" value="0.01" />
	<param name="MPP_XY_VEL_I" type="double" value="0.0" />
	<param name="MPP_XY_VEL_D" type="double" value="0.01" />
	<param name="MPP_XY_VEL_MAX" type="double" value="2.0" />
	<param name="MPP_Z_VEL_P" type="double" value="0.3" />
	<param name="MPP_Z_P" type="double" value="2" />
	<param name="vehicle_model" type="string" value="ardrone" />
</group>

</launch>
